code/modules/projectiles/guns/magic/staff.dm

```diff
@@ -171,6 +171,46 @@
 	process_fire(user, user, FALSE)
 	return FALSE
 
+/**
+ * Staff of chaos given to the wizard upon completing a cheesy grand ritual. Is completely evil and if something
+ * breaks, it's completely intended. Fuck off.
+ * Also can be used by everyone, because why not.
+ */
+/obj/item/gun/magic/staff/chaos/true_wabbajack
+	name = "\proper Wabbajack"
+	desc = "If there is some deity out there, they've definitely skipped their psych appointment before creating this."
+	icon_state = "the_wabbajack"
+	inhand_icon_state = "the_wabbajack"
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF //fuck you
+	max_charges = 999999 //fuck you
+	recharge_rate = 1
+	allow_intruder_use = TRUE
+
+/obj/item/gun/magic/staff/chaos/true_wabbajack/Initialize(mapload)
+	. = ..()
+	allowed_projectile_types |= subtypesof(/obj/projectile/magic)
+	allowed_projectile_types |= subtypesof(/obj/projectile/temp)
+	allowed_projectile_types += subtypesof(/obj/projectile/bullet/rocket)
```

Comment:
why do we have both `|=` and `+=` here

Reply:
because fikou and i'm bad at computer

though we can just use |= everywhere i suppose, as to not dupe the usual chaos staff's pool